include/SDL3/SDL_events.h

```diff
@@ -615,7 +620,27 @@ typedef struct SDL_GamepadButtonEvent
 /**
  * Gamepad device event structure (event.gdevice.*)
  *
+ * Joysticks that are supported Gamepads receive both an SDL_JoyDeviceEvent and
+ * an SDL_GamepadDeviceEvent.
+ *
+ * SDL will populate the event queue with GAMEPAD_ADDED events for joysticks
+ * that are already plugged in duing SDL_Init and are recognized as Gamepads.
+ *
+ * GAMEPAD_ADDED events for controllers plugged in at the start of the program
+ * will not occur automatically if their mappings are added after calling
+ * SDL_Init.
```

Comment:
Yeah, I confirmed that this is not true in SDL3.

Reply:
Awesome, that's great news.